fea-rs/src/compile/compile_ctx.rs

```diff
@@ -1387,7 +1387,7 @@ impl<'a> CompilationCtx<'a> {
                 typed::StatAxisValueItem::Location(loc) => {
                     let loc_tag = loc.tag().to_raw();
                     match loc.value() {
-                        typed::LocationValue::Value(num) => {
+                        typed::StatLocationValue::Value(num) => {
```

Comment:
What's a StatLocationValue? - that name isn't conveying very much to me

Reply:
it's a syntactic construct used only when specifying a STAT table in fea: https://github.com/adobe-type-tools/afdko/blob/develop/docs/OpenTypeFeatureFileSpecification.md#location-format-a-used-in-axis-value-table-format-1-and-format-4